Date: Sun Oct 11 16:14:38 UTC 2020
From: Atri Bhattacharya <badshah400@gmail.com>

Make path for pkgconfig file installation consistent with gRPC_INSTALL_LIBDIR specification

Index: grpc-1.34.0/CMakeLists.txt
===================================================================
--- grpc-1.34.0.orig/CMakeLists.txt
+++ grpc-1.34.0/CMakeLists.txt
@@ -16203,7 +16203,7 @@ function(generate_pkgconfig name descrip
     "${output_filepath}"
     @ONLY)
   install(FILES "${output_filepath}"
-    DESTINATION "lib/pkgconfig/")
+    DESTINATION "${gRPC_INSTALL_LIBDIR}/pkgconfig/")
 endfunction()
 
 # gpr .pc file
